tests/product/standalone/presto_installer.py

```diff
@@ -48,15 +48,26 @@ def get_dependencies():
                 TopologyInstaller]
 
     def install(self, extra_configs=None, coordinator=None,
-                pa_raise_error=True):
+                pa_raise_error=True, rpm_specifier=None):
```

Comment:
Why not pass rpm_specifier into the constructor and handle the logic of deciding what to install there? That way the error checking for _detect_presto_rpm can stay where it was and the error gets raised at its source rather than sometime later.